src/TRE-API/Controllers/SubmissionController.cs

```diff
@@ -352,5 +352,103 @@ public IActionResult SimulateSubmissionProcessing(Submission sub)
                 throw;
             }
         }
+
+        #region ForTest
+
+        /// <summary>
+        /// create secret for minio
+        /// </summary>
+        /// <param name="accessKey">Access Key identifier</param>
+        /// <param name="secretKey">Secret Key identifier</param>
+        [HttpPost("create-minio-secret/{accessKey}/{secretKey}")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        [ProducesResponseType(500)]
+        public async Task<IActionResult> CreateSecretForMinio(
+            [FromRoute] string accessKey,
+            [FromRoute] string secretKey)
+        {
+            try
+            {
+                var result = await _minioSubHelper.CreateUserWithMcAsync(accessKey, secretKey);
+
+                if (result.Success)
+                {
+                    Log.Information("Secret for Minio Access Key {accessKey}, Secret Key {secretKey}", accessKey, secretKey);
+                    return Ok(new { Message = "Secret Created successfully" });
+                }
+
+                return NotFound(new { Message = "Failed to create secret for user" });
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error creating secret for Access Key {accessKey}, Secret Key {secretKey}", accessKey, secretKey);
+                return StatusCode(500, new { Message = "Internal server error", Error = ex.Message });
+            }
+        }
+
+        /// <summary>
+        /// Set Policty to a secret
+        /// </summary>
+        /// <param name="accessKey">Access Key identifier</param>
+        /// <param name="policyName">Secret Key identifier</param>
+        [HttpPost("set-policy-minio-secret/{accessKey}/{policyName}")]
```

Comment:
As above, not sure this should be on SubmissionController, and it shouldn't explicitly reference minio unless it's not intended to be part of the standard API.